src/pre-commit.check_working_on_master.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+BRANCH_MASTER="master";
+
+if [[ $(git rev-parse --abbrev-ref HEAD) == $BRANCH_MASTER ]];
+then
+    echo "!!!WARNING!!! Now you working on the master branch."
```

Comment:
Let's add some color :)

Change this line to `printf "\n$YELLOW!!!WARNING!!! Now you working on the master branch.\n$CO"`, and don't forget to add constants with color-code at top of this script

```bash
YELLOW='\033[0;33m'
CO='\033[0m'
```

Reply:
Ok. This is a good idea:) Added.